wolfcrypt/src/aes_gcm_asm.asm

```diff
@@ -1960,15 +1980,25 @@ AES_GCM_decrypt PROC
         mov	rsi, rdx
         mov	r12, r8
         mov	rax, r9
-        mov	r8, QWORD PTR [rsp+104]
-        mov	r9d, DWORD PTR [rsp+112]
-        mov	r11d, DWORD PTR [rsp+120]
-        mov	ebx, DWORD PTR [rsp+128]
-        mov	r14d, DWORD PTR [rsp+136]
-        mov	r15, QWORD PTR [rsp+144]
-        mov	r10d, DWORD PTR [rsp+152]
-        mov	rbp, QWORD PTR [rsp+160]
-        sub	rsp, 168
+        mov	r8, QWORD PTR [rsp+264]
```

Comment:
again why +264 ?

Reply:
As above, stack used for temporary storage.